sass/components/_popover.scss

```diff
@@ -72,6 +72,7 @@
     &.left {
         > .arrow {
             display: none;
+            border-left-color: rgba(var(--center-channel-color-rgb), 0.25);
```

Comment:
This is adding a border-left-color to elements that didn't have it before since only `.left > .arrow` had it before, but I'm not sure if that actually causes any problems here.

@asaadmahmood Are you aware of why we specified different borders for each version of the arrow before? Perhaps we could just set `border-color` for all of them at once.

If we can't do that, then this should be moved into its own rule to only target `.left > .arrow`

Reply:
The CSS style is now moved to the correct place, only targeting `.left > .arrow`. I can also change it to set `border-color` once if that's preferred!

Reply:
I think it's safe to leave it as-is. I'm a bit worried about changing to just `border-color` since this is quite old CSS. Thanks for moving it though!